crates/matrix-sdk/src/room/mod.rs

```diff
@@ -1468,6 +1468,7 @@ impl Room {
     /// [`SyncMessageLikeEvent`]: ruma::events::SyncMessageLikeEvent
     /// [`StateUnsigned`]: ruma::events::StateUnsigned
     /// [`transaction_id`]: ruma::events::StateUnsigned#structfield.transaction_id
+    #[instrument(skip(self, content), fields(room_id = %self.room_id(), transaction_id, encrypted))]
```

Comment:
We could skip `txn_id` if `transaction_id` is more complete later?

Reply:
Right, fixed in https://github.com/matrix-org/matrix-rust-sdk/pull/2405/commits/b38b675fe7ef9769d432f07cad142ff82f721e58.